rust/processor/src/db/postgres/migrations/2025-03-12-000346_add_latest_arena_candlesticks_view/up.sql

```diff
@@ -0,0 +1,8 @@
+-- Your SQL goes here
+
+-- The latest arena candlesticks, distinct on each `period`.
+CREATE VIEW arena_latest_candlesticks AS (
+    SELECT DISTINCT ON (melee_id, period) *
+    FROM arena_candlesticks
+    ORDER BY melee_id, period, start_time DESC
+);
```

Comment:
```suggestion
-- Your SQL goes here

-- The latest arena candlesticks, distinct on each `period`.
CREATE VIEW arena_latest_candlesticks AS (
    SELECT DISTINCT ON (melee_id, period) *
    FROM arena_candlesticks
    ORDER BY melee_id, period, start_time DESC
);
```

We don't actually need this because we don't ahve the same constraints as the original contract. That is, the latest candlestick in the candlesticks endpoint IS actually the "current" candlestick for the arena candlesticks, whereas with the original contract it wasn't